mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/VirksomhetRoutes.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.mulighetsrommet.api.routes.v1
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.mulighetsrommet.api.clients.enhetsregister.AmtEnhetsregisterClient
+import org.koin.ktor.ext.inject
+
+fun Route.virksomhetRoutes() {
+    val amtEnhetsregisterClientImpl: AmtEnhetsregisterClient by inject()
+    route("api/v1/internal/virksomhet") {
+        get("{orgnr}") {
+            val orgnr = call.parameters["orgnr"]?.toInt() ?: return@get call.respondText(
```

Comment:
Dette har ikke jeg lagt merke til tidligere, men orgnr. burde vel være string i stedet for int (antagelig også i enhetsregisterClient også), blir vel fort en bug om vi har vikrsomhetsnummere som begynner med `0`.
Kan evt. gjøres som en egen oppgave?

Reply:
Ja, det er et godt poeng. Jeg stusset på det i går, men lot det være. Jeg ser på hvor mye endringer som kreves for å endre til string allerede nå.